test/datasets/containers.jl

```diff
@@ -0,0 +1,53 @@
+include("imports.jl")
+
+@testset ExtendedTestSet "TableDataset" begin
+
+    @testset ExtendedTestSet "TableDataset from rowaccess table" begin
+        Tables.columnaccess(::Type{<:Tables.MatrixTable}) = false
+        Tables.rowaccess(::Type{<:Tables.MatrixTable}) = true
+        
+        testtable = Tables.table([1 4.0 "7"; 2 5.0 "8"; 3 6.0 "9"])
+        td = TableDataset(testtable)
+
+        @test [data for data in getobs(td, 1)] == [1, 4.0, "7"]
```

Comment:
Not sure if broadcasting works here:
```suggestion
        @test all(getobs(td, 1) .== [1, 4.0, "7"])
```

Reply:
It should work for `MatrixRow` objects.